content/scala-cli.md

```diff
@@ -146,14 +146,14 @@ For example, if the syntax of the language is changed, this section should list
 
 ### Compatibility
 
-Adopting Scala CLI as new `scala` command as is will change some behaviours of current scripts. Some examples:
+Adopting Scala CLI as the new `scala` command, as is, will change some of the behaviour of today's scripts. Some examples:
 
-- `scala repl`  needs to run to start `repl` instead of `scala`
-- with `scala compile` and `scala doc` some of the more obscure (or brand new) compile options will needs to be prefixed with `-o`
-- Scala CLi recognizes tests based on the extension used (`*.test.scala`) so running `scala compile a.scala a.test.scala` will only compile `a.scala`
-- Scala CLI has its own versioning scheme, that is not related to Scala compiler. Default version used may dynamically change when new Scala version is released.
-- By default, Scala CLI manages its own dependencies (e.g. scalac, zinc, bloop) and resolved them lazily. This means that first run after Scala CLI is installed will resolve quite some dependencies. Moreover, Scala CLI periodically check for updates, new defaults accessing online resources (but it is not required to work, so Scala CLI can work in offline environment once setup)
-- Scala CLI can be also configured using driectives. Command line options has precendece over using directives, however using directives overrides the default. Compiling file starting with following line `//> using scala 2.13.8` without provided version in command line will result using `2.13.8` rather then default version. We consider this a feature however technially is a breaking change.
+- `scala repl` needs to be run to start a REPL instead of just `scala`
+- with `scala compile` and `scala doc`, some of the more obscure (or brand new) compile options need to be prefixed with `-O`
+- Scala CLI recognizes tests based on the extension used (`*.test.scala`) so running `scala compile a.scala a.test.scala` will only compile `a.scala`
+- Scala CLI has its own versioning scheme, that is not related to the Scala compiler. Default version used may dynamically change when new Scala version is released.
+- By default, Scala CLI manages its own dependencies (e.g. scalac, zinc, Bloop) and resolves them lazily. This means that the first run of Scala CLI resolves quite some dependencies. Moreover, Scala CLI periodically checks for updates, new defaults accessing online resources (but it is not required to work, so Scala CLI can work in offline environment once setup)
+- Scala CLI can also be configured via using directives. Command line options have precedence over using directives, however using directives overrides defaults. Compiling a file starting with `//> using scala 2.13.8`, without providing a Scala version on the command line, will result in using `2.13.8` rather than the default Scala version. We consider this a feature. However, technically, this is a breaking change.
```

Comment:
```suggestion
- Scala CLI can also be configured via using directives. Command line options have precedence over using directives, however using directives override defaults. Compiling a file starting with `//> using scala 2.13.8`, without providing a Scala version on the command line, will result in using `2.13.8` rather than the default Scala version. We consider this a feature. However, technically, this is a breaking change.
```